bonnie-rest/src/main/java/com/cgi/hexagon/bonnierest/UserController.java

```diff
@@ -32,6 +34,40 @@ public ResponseEntity<User> getUser(@PathVariable long id) {
         }
     }
 
+    @GetMapping("/get/current/name")
```

Comment:
[05/10 14:17] Szanics, Levente
One thing that triggers my OCD: Do not use "get" in the path. It is not RESTish. This is a bad habit from SOAP. Just forget about it. Use GET "/api/order" to get all orders, use POST "/api/order" to create a new order, use GET "/api/order/{orderId}" to get the order with id "orderId".
 :-)